### PR TITLE
Blogged: Alert notifiers for mobile devices

### PR DESCRIPTION
**Pull request by tsegismont**
Blogged: Alert notifiers for mobile devices

### CODE REVIEW

**Comment by tsegismont** [comment]
@pilhuhn can you review?

File: src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc [hunk 1]
```diff
@@ -0,0 +1,72 @@
+= Alert notifiers for mobile devices
+Thomas Segismont
+2015-04-14
+:jbake-type: post
+:jbake-status: draft
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc:5]: Why draft?

File: src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+:jbake-tags: blog, alerts, mobile, SMS, Aerogear
+:idprefix:
+:linkattrs:
+
+Hawkular Alerts has new notifications senders which target mobile devices. Let's start with a brief explanation of
+notification sending.
+
+== How notification senders work?
+
+When the alert engine detects a problem, it sends an action message to the configured sender(s) for this type issue. The
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc]: type _of_ issue?

File: src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc (hunk 1 above)
Comment (tsegismont) [inline src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc:5]: Le 14/04/2015 11:38, Heiko W. Rupp a écrit :

> Why draft?

In order to see how it looks online before publishing.

File: src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc (hunk 2 above)
Comment (tsegismont) [inline src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc]: Le 14/04/2015 11:39, Heiko W. Rupp a écrit :

> type /of/ issue?

yes, thanks

File: src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+authorization `token`, and `from` number.
+
+=== More to come...
+
+We have planned to integrate with http://www.pagerduty.com/[PagerDuty]. In the beginning, each notification message
+will be sent with a new indentifier. As a consequence, PagerDuty will see recurrent notifications of the same problem
+as different incidents. The Hawkular Alerts team is working on an incident worflow and we should be able to solve this
+problem in the near future.
+
+That's it! Questions? Comments? Feel free to ask here or join in `#hawkular` channel on Freenode!
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc]: Make it a irc://irc.freenode.net/#hawkular link

File: src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+endif::[]
+
+The application lacks proper formatting of course, but you can see on this screenshot that the notification message
+landed properly on the mobile device after travelling through the Hawkular Bus, the Aerogear Unified Push Server and
+Google Cloud's Messaging platform.
+
+=== Short Message Service
+
+There's also an `sms` plugin which is built on https://www.twilio.com/sms[Twilio's SMS services]. No mobile application
+required here, all you need to do is to create a Twilio account and configure the plugin with your application `sid`,
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc:83]: Can you describe / link to docs how to configure the plugin(s)?

File: src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc (hunk 3 above)
Comment (tsegismont) [inline src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc]: Le 14/04/2015 11:41, Heiko W. Rupp a écrit :

> Make it a irc://irc.freenode.net/#hawkular link

Sure

File: src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc (hunk 4 above)
Comment (tsegismont) [inline src/main/jbake/content/blog/2015/04/09/alert-notifiers-for-mobile-devices.adoc:83]: Le 14/04/2015 11:42, Heiko W. Rupp a écrit :

> Can you describe / link to docs how to configure the plugin?

I'll add a short description

**Pull request merged**